File: backend/database/connection.js
const { Client } = require('pg');

const sql = new Client({
  user: 'postgres',
  host: 'localhost',
  database: 'exchanges',
  password: 'postgre',
  port: 5432,
});

sql.connect((err) => {
  if (err) return console.log(err);
  console.log('Connected to DB');
});

exports.sql = sql;

exports.endConnection = () => {
  setTimeout(() => {
    sql
      .end()
      .then(() => console.log('Disconnected DB'))
      .catch(err => console.error('Error during disconnection', err.stack));
  }, 1000);
};

File: backend/forecast/connection.py
import psycopg2

PGHOST = 'localhost'
PGDATABASE = 'exchanges'
PGUSER = 'postgres'
PGPASSWORD = 'postgre'
PGPORT = '5432'

conn_string = "host=" + PGHOST + " port=" + PGPORT + " dbname=" + \
    PGDATABASE + " user=" + PGUSER + " password=" + PGPASSWORD

conn = psycopg2.connect(conn_string)
print("Connected!")
